src/app/components/modules/LoginForm.scss

```diff
@@ -7,18 +7,18 @@
   form {
     margin-top: 1.5rem;
   }
-  .button.sign-up {
+  .sign-up {
+    .button {
```

Comment:
Nitpick - wondering about the rational for making this a class `.button` & not just an element type `button`, particularly when the class name IS the element name (lots of possibility for collision as css changes over time)

Reply:
left it as-is because I figured it was inheriting from some global .button class